sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/WritableColumnVector.java

```diff
@@ -763,6 +805,10 @@ protected boolean isArray() {
       DecimalType.isByteArrayDecimalType(type);
   }
 
+  protected boolean isStruct() {
+    return type instanceof StructType || type instanceof CalendarIntervalType;
```

Comment:
Do you know if we have tests for this? I am curious what happens if someone adds a union type or another composite type, will we need to update this method?

Reply:
Hmm this is no longer used - I created it for the `WritableColumnVector` change I think, but now the change is no longer needed. Let me remove this.